community/blog/2022-04 Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6/README.md

```diff
@@ -0,0 +1,105 @@
+# Finished Azure Integration Account functionality and other cool features in Arcus Scripting v0.6
+Recently, we have released Arcus Scripting v0.6. The scripting library was quiet for a while, without any updates, so let's change that with this run-down post of new functionality coming to Arcus Scripting.
+
+## Upload all Azure Integration Account artifacts
+Azure Integration Accounts are critical for business-to-business (B2B) and integration workflows with Azure Logic Apps. The communication between different parties often requires artifacts. These artifacts include [trading partners](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-partners), [agreements](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-agreements), [maps](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-maps), [schemas](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-maps), [certificates](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-certificates)...
```

Comment:
```suggestion
Azure Integration Accounts are critical for business-to-business (B2B) and integration workflows with Azure Logic Apps. The communication between different parties often requires artifacts. These artifacts include [trading partners](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-partners), [agreements](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-agreements), [maps](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-schemas), [schemas](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-maps), [certificates](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-certificates)...
```

Reply:
Oh, I see, it's a link that's wrong, but an other one that's proposed here 😄 .

Reply:
ah my bad